Slim/Utils/Misc.pm

```diff
@@ -1435,6 +1436,28 @@ sub min {
   return $m;
 }
 
+=head2 obfuscate ( )
```

Comment:
I think you can remove these, as we're not storing any credentials?

Reply:
I was thinking of keeping them here as a general services (I use it in my plugins) and maybe if we can think of a better way to implement that in the future, that would a good thing to have a centralized service

Reply:
I just don't like to keep code around which is not being used by LMS itself. It risks to be ripped out as "unused" at some point...

At least call it `deobfuscate`, and add a note that these are there as a service for plugins.

Reply:
If it really bothers you, I can remove it and duplicate it in the plugins

Reply:
remove